dss/api/bundles/__init__.py

```diff
@@ -187,8 +189,38 @@ def enumerate(replica: str,
     return response
 
 
+def project_uuid_from_file(replica, file):
```

Comment:
This method adds implicit metadata requirements to the DSS. The DSS should remain metadata agnostic.